src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -26,18 +22,24 @@ public class StandbyManager : IStandbyManager
         private readonly IScriptHostManager _scriptHostManager;
         private readonly IOptionsMonitor<ScriptApplicationHostOptions> _options;
         private readonly Lazy<Task> _specializationTask;
+        private readonly IScriptWebHostEnvironment _environment;
         private readonly IConfigurationRoot _configuration;
         private readonly ILogger _logger;
+
+        private readonly TimeSpan _specializationTimerInterval = TimeSpan.FromMilliseconds(500);
```

Comment:
This used to be 1 second. I lowered it down as this check is cheap and can affect cold start. We may be able to lower it even further.

Reply:
This is background coldstart though, not http triggered scenerios where it matters most.

Reply:
Aren't there _some_ HTTP scenarios that end up relying on this? Don't we have a dependency on this in some easy auth scenarios where the host never gets the ping?

Reply:
Yes, you're right, as described in https://github.com/Azure/azure-functions-host/issues/2201.